bash_command_line_102/bash_command_line_102.md

```diff
@@ -482,12 +482,12 @@ rm brown_bear.txt
 <div class = "warning">
 <b style="color: rgb(var(--color-highlight));">Warning!</b><br>
 
-Removing files from your command line interface with the `rm` command **removes** them from your computer. It does not move them to your "Trash" folder.
+Removing files via your command line interface with the `rm` command **removes** them from your computer. It does not move them to your "Trash" folder.
```

Comment:
```suggestion
Removing files via your command line interface with the `rm` command **permanently removes** them from your computer. It does not move them to your "Trash" folder.
```

adding permanently might also drive this point home.

Reply:
That's the word I was looking for! tysm!!